commerce_coordinator/apps/lms/views.py

```diff
@@ -95,24 +74,22 @@ def _redirect_response_payment(self, request):
         return redirect
 
     @staticmethod
-    def _add_utm_and_waffle_flags_params_to_url(url, params):
+    def _add_query_params_to_redirect_url(url, params):
         """
-        Add UTM (Urchin Tracking/Google Analytics) flags to the URL for the MFE to use in its reporting,
-        as well as any Waffle Flags
-
+        Add UTM (Urchin Tracking/Google Analytics) flags to the URL for the MFE to use in its reporting
         Args:
             params (list): Query Params from Req as an encoded list
         Returns:
             url (str): A URL asa Python String
         """
 
-        # utm_params is [(u'utm_content', u'course-v1:IDBx IDB20.1x 1T2017'),... + waffles
-        utm_params = [item for item in params
-                      if QueryParamPrefixes.GOOGLE_ANALYTICS.value or QueryParamPrefixes.WAFFLE_FLAG.value in item[0]]
+        # utm_params is [(u'utm_content', u'course-v1:IDBx IDB20.1x 1T2017'),...
+        query_params = [item for item in params
```

Comment:
My previous suggestion was to add a new utility and keep this one as it is since we were not going to use it. My expectation was that in our new utility we would just append the query params that we received from the track selection page as it is. We don't need to do any conditional check as far as I understand nor do we need to any additional params(e.g. google analytics etc). We just send what we receive as it is(which would just be the sku and course_run_key IMO).
That is my understanding.

Reply:
I had a discussion with @shafqatfarhan and we agreed that rather than adding a new utility function, we can modify this current one to add all query params to the redirect url. Reason being, all the previous code in this file is no longer needed hence modifying this endpoint to satisfy our needs.